scripts/initializers/company-switcher.js

```diff
@@ -0,0 +1,107 @@
+import { events } from '@dropins/tools/event-bus.js';
+import { getHeaders, getConfigValue } from '@dropins/tools/lib/aem/configs.js';
+import { initializers } from '@dropins/tools/initializer.js';
+import {
+  initialize,
+  setFetchGraphQlHeaders,
+  setFetchGraphQlHeader as setCompanyGraphQlHeader,
+  removeFetchGraphQlHeader as removeCompanyGraphQlHeader,
+} from '@dropins/storefront-company-switcher/api.js';
+import {
+  setFetchGraphQlHeader as setCatalogGraphQlHeader,
+  removeFetchGraphQlHeader as removeCatalogGraphQlHeader,
+} from '@dropins/storefront-pdp/api.js';
+import {
+  setFetchGraphQlHeader as setCartGraphQlHeader,
+  removeFetchGraphQlHeader as removeCartGraphQlHeader,
+} from '@dropins/storefront-cart/api.js';
+import {
+  setFetchGraphQlHeader as setSearchGraphQlHeader,
+  removeFetchGraphQlHeader as removeSearchGraphQlHeader,
+} from '@dropins/storefront-product-discovery/api.js';
+import {
+  setFetchGraphQlHeader as setOrderGraphQlHeader,
+  removeFetchGraphQlHeader as removeOrderGraphQlHeader,
+} from '@dropins/storefront-order/api.js';
+import {
+  setFetchGraphQlHeader as setAccountGraphQlHeader,
+  removeFetchGraphQlHeader as removeAccountGraphQlHeader,
+} from '@dropins/storefront-account/api.js';
+
+import { initializeDropin } from './index.js';
+import { fetchPlaceholders } from '../commerce.js';
+
+const headerKey = getConfigValue('company-switcher.company-header-key');
+const companySessionKey = getConfigValue('company-switcher.company-session-key');
+
+const setCompanyHeaderFns = [
+  setCompanyGraphQlHeader,
+  setCatalogGraphQlHeader,
+  setCartGraphQlHeader,
+  setSearchGraphQlHeader,
+  setOrderGraphQlHeader,
+  setAccountGraphQlHeader,
+];
+
+const removeCompanyHeaderFns = [
+  removeCompanyGraphQlHeader,
+  removeCatalogGraphQlHeader,
+  removeCartGraphQlHeader,
+  removeSearchGraphQlHeader,
+  removeOrderGraphQlHeader,
+  removeAccountGraphQlHeader,
+];
```

Comment:
Just a suggestion: What about importing the `fetchGraphQl` instance of each since they all share the same methods? 

```js
import * as pdpFetchGraphQl from '@dropins/storefront-pdp/api.js';
import * as cartFetchGraphQl from '@dropins/storefront-cart/api.js';
...

[
  pdpFetchGraphQl,
  cartFetchGraphQl.
  ...
].forEach(({ setFetchGraphQlHeader,  removeFetchGraphQlHeader }) => {
  setCompanyHeaderFns(setFetchGraphQlHeader)
  removeFn(removeFetchGraphQlHeader);
})
```